src/content/1.8/functoriality.tex

```diff
@@ -27,7 +27,7 @@ \section{Bifunctors}
 morphisms, one from $\cat{C}$ and one from $\cat{D}$, to a morphism in $\cat{E}$.
 
 Again, a pair of morphisms is just a single morphism in the product
-category $\cat{C}\times{}\cat{D}$ to $\cat{E}$. We define a morphism in a Cartesian product of categories
+category $\cat{C}\times{}\cat{D}$. We define a morphism in a Cartesian product of categories
```

Comment:
Why this change ?

Reply:
This is already fixed in the [original blog post](https://bartoszmilewski.com/2015/02/03/functoriality/). I think this is just a typo. If my understanding is correct, the single morphism doesn't go from the category (C x D) to another category (E). It is not a functor.